app/client/components/payment/update/card/stripeCardInputForm.tsx

```diff
@@ -110,9 +119,9 @@ export const StripeCardInputForm = (props: StripeCardInputFormProps) => {
         return;
       }
 
-      //
       const intentResult = await stripe.confirmCardSetup(
-        props.stripeSetupIntent.client_secret
+        props.stripeSetupIntent.client_secret,
+        { payment_method: createPaymentMethodResult.paymentMethod.id }
```

Comment:
on reflection this explicitness is better than the implicit way this worked before 👍